about_solid.html

```diff
@@ -100,15 +100,15 @@ <h3>Solid Servers and Pods</h3>
                         </p>
                         <p>Each Pod is fully controlled by the Pod owner (i.e., you).
                             And each Pod's data and access rules are fully distinct from those of other Pods.</p>
-                        <p>You can get a Pod from a Pod Provider, or you may choose to self-host your Pod.</p>
-                        <p>You can even have multiple Pods. They can be hosted by the same Pod Provider or by
-                            different Providers or be self-hosted or any combination thereof. The number of Pods you
-                            have as well as which Solid Server or Servers you use is effectively transparent to the
-                            applications and services that you use. This is because, in the Solid ecosystem, data is
+                        <p>People can get a Pod from a Pod Provider, or you may choose to self-host your Pod.</p>
+                        <p>People can even have multiple Pods. They can be hosted by the same Pod Provider or by
+                            different Providers or be self-hosted or any combination thereof. The number of Pods someone
+                            has, as well as which Solid Server or Servers they use, is effectively transparent to the
+                            applications and services that they use. This is because, in the Solid ecosystem, data is
```

Comment:
```suggestion
                        <p>An entity can get a Pod from a Pod Provider, or they may choose to self-host their Pod.</p>
                        <p>Entities can even have multiple Pods. They can be hosted by the same Pod Provider or by
                            different Providers or be self-hosted or any combination thereof. The number of Pods an entity
                            has, as well as which Solid Server or Servers they use, is effectively invisible to the
                            applications and services that they use. This is because, in the Solid ecosystem, data is
```